tests/visualization_tests/matplotlib_tests/test_pareto_front.py

```diff
@@ -1,14 +1,24 @@
 import itertools
 from typing import List
 from typing import Optional
+from typing import Union
 
+from matplotlib.collections import PathCollection
 import numpy as np
 import pytest
 
 import optuna
 from optuna.visualization.matplotlib import plot_pareto_front
 
 
+def allclose_as_set(
+    points1: Union[List[List[float]], np.ndarray], points2: Union[List[List[float]], np.ndarray]
+) -> bool:
+    p1 = points1 if type(points1) == list else points1.tolist()  # type: ignore
+    p2 = points2 if type(points2) == list else points2.tolist()  # type: ignore
```

Comment:
Why do you need to ignore the mypy checking here? We should chech with mypy whenever we can.

Reply:
I found `isinstance` in `if` can handle `Uniont` type better and removed `# type: ignore`.
https://github.com/optuna/optuna/pull/3183/commits/08c77aac4141b0f3ccc0a2a9da51557a53cdabe5